### PR TITLE
Fixed README text

### PR DESCRIPTION
**Pull request by eugeneyche**
Hey, can someone review this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # dragonslave
-Lina lover's only
+Lina lovers only
```
Comment (jytang) [inline README.md:2]: Great change! The possessive case of "lover" did not work here because "only" is an adjective.

Comment (kitadak) [inline README.md:2]: Is this missing end punctuation?

Comment (eugeneyche) [inline README.md:2]: I personally think more memes need to be in it, though :tada:

**Comment by eugeneyche** [comment]
@addtea it seems to me that a lot of people, including myself, have adopted the convention of not ending commits with periods (fyi)

**Pull request merged**